src/cljs/main/broadfcui/page/workspace/summary/acl_editor.cljs

```diff
@@ -72,7 +72,11 @@
       [:div {:style {:margin "0.5rem 0"}}
        [comps/Button {:text "Add new" :icon :add
                       :onClick #(swap! state update :non-project-owner-acl-vec
-                                       conj {:email "" :accessLevel "READER" :canShare false})}]]
+                                       conj (let [permissions {:email "" :accessLevel "READER"}]
+                                              ; Only owners can set new canShare permissions, so we only want to include
+                                              ; those in the default settings when the user is at least an owner
+                                              (if (common/access-greater-than-equal-to? user-access-level "OWNER")
+                                                (conj permissions (:canShare false)) permissions)))}]]
```

Comment:
`(conj permissions (:canShare false))` should be `(assoc permissions :canShare false)`.  Then move the "else" case to the next line for clarity.